azurerm/internal/services/logic/data_source_logic_app_workflow.go

```diff
@@ -51,12 +51,23 @@ func dataSourceArmLogicAppWorkflow() *schema.Resource {
 				Computed: true,
 			},
 
-			"tags": tags.SchemaDataSource(),
-
 			"access_endpoint": {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"endpoint_configuration": {
```

Comment:
@magodo, would this schema perhapes make more sense as:
```
			"endpoint_ip_addresses": {
				Type:     schema.TypeList,
				Computed: true,
				Elem: &schema.Resource{
					Schema: map[string]*schema.Schema{
						"workflow":  buildFlowEndpointsSchema(),
						"connector": buildFlowEndpointsSchema(),
					},
				},
			},

func buildFlowEndpointsSchema() *schema.Schema {
	return &schema.Schema{
		Type:     schema.TypeList,
		Computed: true,
		Elem: &schema.Resource{
			Schema: map[string]*schema.Schema{
				"outgoing": {
					Type:     schema.TypeSet,
					Computed: true,
					Elem:     &schema.Schema{Type: schema.TypeString},
				},
				"access_endpoint": {
					Type:     schema.TypeSet,
					Computed: true,
					Elem:     &schema.Schema{Type: schema.TypeString},
				},
			},
		},
	}
}

````

Reply:
wdyt?

Reply:
@katbyte Thank you for reviewing my PR, regarding to this one, I feel like we'd better to reflect tha they are both ip address and that property to be an collection type. So I would prefer the original naming...

Reply:
taking a look at this these want to become 4 top-level fields:

```
"connector_endpoint_ip_addresses": {
	Type:     schema.TypeList,
	Computed: true,
	Elem:     &schema.Schema{Type: schema.TypeString},
},
"connector_outbound_ip_addresses": {
	Type:     schema.TypeList,
	Computed: true,
	Elem:     &schema.Schema{Type: schema.TypeString},
},
"workflow_endpoint_ip_addresses": {
	Type:     schema.TypeList,
	Computed: true,
	Elem:     &schema.Schema{Type: schema.TypeString},
},
"workflow_outbound_ip_addresses": {
	Type:     schema.TypeList,
	Computed: true,
	Elem:     &schema.Schema{Type: schema.TypeString},
},
```

Note these'll need to be lists in any case to be fully accessible